src/ProjectManager.cpp

```diff
@@ -270,9 +269,9 @@ void InitProjectWindow( ProjectWindow &window )
    //      will be given the focus even if we try to SetFocus().  By
    //      making the TrackPanel that first window, we resolve several
    //      keyboard focus problems.
-   pPage->MoveBeforeInTabOrder(topPanel);
+   window.GetContainerWindow()->MoveAfterInTabOrder(topPanel);
```

Comment:
Tab order reminds me that we may need to check out how tabbing now works in the presence of the new window.  @DavidBailes might have opinions.